transfer/src/main/res/values/strings.xml

```diff
@@ -3,4 +3,6 @@
     <string name="receiver_activity_title">Kin Ecosystem</string>
     <string name="receiver_activity_agree">I agree</string>
     <string name="receiver_activity_message">In order to send Kin, %1$s would like to receive your Kin account information (i.e. your public address) from %2$s.</string>
+    <string name="discover_title_no_data">Oh… Something isn\’t working</string>
+    <string name="discover_subtitle_no_data">An unknown error occurred. Please close the page and try again later.</string>
```

Comment:
I'd say these ^^ are generic errors - would rename the key to `error_no_data_title`, `error_no_data_subtitle`

Reply:
cool